schema/sm-specific-schema-patches/scopy.sh

```diff
@@ -27,7 +27,7 @@ function find_source ()  {
     local db=""
 
     local imp_packages=(rhn_entitlements.pkb rhn_channel.pks rhn_channel.pkb rhn_server.pkb)
-    local imp_procs=(pg_dblink_exec create_first_org create_new_org)
+    local imp_procs=(create_first_org create_new_org)
 
     if [[ "$s" == *oracle ]]; then
 	db="oracle"
```

Comment:
Can't we kick those 2 lines out?

Reply:
Whole `schema/sm-specific-schema-patches` seems to be obsolete and can be dropped, IMHO. It has not been touched since 2018. But that is for another PR.

Reply:
Yes, this was needed when spacewalk had incompatible changes compared to SUMA and we had to copy them again and again over. We can remove this